src/java/org/apache/cassandra/db/ExpirationDateOverflowHandling.java

```diff
@@ -49,7 +51,7 @@
     public static ExpirationDateOverflowPolicy policy;
 
     static {
-        String policyAsString = System.getProperty("cassandra.expiration_date_overflow_policy", ExpirationDateOverflowPolicy.REJECT.name());
+        String policyAsString = EXPIRATION_DATE_OVERFLOW_POLICY.getString(ExpirationDateOverflowPolicy.REJECT.name());
```

Comment:
can not be the default value moved to `CassandraRelevantProperties#EXPIRATION_DATE_OVERFLOW_POLICY` ?

Reply:
Yes, it is possible. However, the `CassandraRelevantProperties` seems to be widely used (by different modules or applications) thus I think it is better to avoid adding extra class dependencies to this class. From my understanding, it will be better to avoid having any defaults in the `CassandraRelevantProperties`, but this discussion is not for this thread I think.

I you think we should move it, I'll do it.

Reply:
ok keep it there